src/renderer/js/component/form.js

```diff
@@ -34,16 +35,10 @@ export class Form extends React.PureComponent {
 }
 
 export class FormRow extends React.PureComponent {
-  static propTypes = {
-    label: React.PropTypes.oneOfType([
-      React.PropTypes.string,
-      React.PropTypes.element,
-    ]),
-    errorMessage: React.PropTypes.oneOfType([
-      React.PropTypes.string,
-      React.PropTypes.object,
-    ]),
-    // helper: React.PropTypes.html,
+  spropTypes = {
```

Comment:
@seanyesmunt I think this got merged with a typo

Reply:
👍